src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -80,6 +96,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if (null !== $label && '' !== trim($label)) {
             $options = array('graph' => array('label' => $label));
         }
+        $options += array('name' => $serviceId);
```

Comment:
Could you use array_replace? (more "symfony style")

Reply:
done